### DISCUSSION TITLE
omz_history exectued with each command

### DISCUSSION THREAD

**Discussion by MohamedElashri** [ohmyzsh/ohmyzsh#11450] [Help] [answered]
I'm getting something like

```
[1]    7514 done       omz_history -D -t "%s" -1 |
       7516 killed
```
with each command or start of my zsh environment. I could not trace thar problem. any suggestion?

**Comment by oibafs** [comment, marked as answer]
Do you use ShellHistory app? I was getting the same problem because I had an expired beta. After reinstalling it from AppStore the problem is gone.

**Comment by mcornella** [comment]
It's possible something is calling `history`, perhaps on a precmd hook function.

We could see this with command `which $precmd_functions`, please post the output of that.
